learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Analytics
+description: Meilisearch Cloud offers in-depth search analytics to help you understand how users search in your application.
+sidebarDepth: 3
+---
+
+# Analytics
+
+Enable Meilisearch Cloud analytics to gain insight on what your users are searching for, how often they find it, and how much they have to wait between typing a query and receiving results.
+
+## Requirements
+
+You must have a Meilisearch Cloud account to have access to search analytics.
+
+## Activating analytics
+
+### Enable analytics in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics" section and click on the "Enable analytics" button:
+
+![The analytics section of the project overview. It shows one button, "Enable analytics", and a short explanation of the feature.](https://raw.githubusercontent.com/meilisearch/documentation/cloud-analytics-guide/assets/images/cloud-analytics/01-analytics-enable.png)
+
+Meilisearch Cloud will begin processing your request. The "Analytics" section will update when the feature is enabled.
+
+### Update URLs in your application
+
+Enabling analytics generates a new URL for your project. Meilisearch Cloud is only able to track metrics for queries sent to this URL. 
+
+Update your application so all API requests point to the new URL:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com/indexes/products/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "green socks" }'
+```
+
+Your old project URL will remain functional, but API calls using the old URL will not be reflected in the analytics interface.
+
+<Capsule intent="note" title="Analytics and latency">
+To capture and process search analytics data, Meilisearch Cloud must route API requests through a proxy. This might increase latency in some applications.
+</Capsule>
+
+### Configuring click-through rate and average click position
+
+To track metrics like clickthrough rate and average click position, Meilisearch Cloud needs to know when users click on search results.
```

Comment:
clickthrough is spelled differently here than in the title above.

"click-through" seems to be the spelling Grammarly prefers

Reply:
I would add when and where (since the position of the search result matters in the final number)

Reply:
@Strift: thanks for spotting it. "Click-through" is indeed the only correct spelling at the moment. Most compound nouns tend to become a single word with time, but we're not there yet 😞